packages/storage/amplify_storage_s3_dart/example/bin/example.dart

```diff
@@ -508,6 +515,17 @@ StorageAccessLevel promptStorageAccessLevel(String message) {
   return accessLevel;
 }
 
+bool promptUseAcceleration() {
+  String input;
+
+  do {
+    input = prompt('Use transfer acceleration for this operation? (Y/n): ')
+        .toLowerCase();
+  } while (input != 'y' && input != 'n');
```

Comment:
`(Y/n)` implies that an empty response == 'Y'.
`(y/n)` would imply no default value.